src/components/CheckForm/CheckForm.tsx

```diff
@@ -166,22 +168,11 @@ export const CheckForm = ({ check, disabled, pageTitle }: CheckFormProps) => {
             >
               {!isExistingCheck && <OverLimitAlert checkType={checkType} />}
 
-              {checkType === CheckType.Browser && (
-                <Alert severity="info" title="">
-                  <div>
-                    Browser checks are in private preview. During the preview they are free to use: test executions will
-                    not be billed.{' '}
-                    <TextLink
-                      href="https://grafana.com/docs/grafana-cloud/cost-management-and-billing/understand-your-invoice/synthetic-monitoring-invoice/"
-                      external={true}
-                    >
-                      Read more
-                    </TextLink>
-                  </div>
-                </Alert>
-              )}
-
-              <FormLayout.Section label={checkTypeStep1Label[checkType]} fields={[`job`, ...defineCheckFields]}>
+              <FormLayout.Section
+                label={checkTypeStep1Label[checkType]}
+                fields={[`job`, ...defineCheckFields]}
+                status={status}
```

Comment:
I found the alert banner quite overwhelming and distracting from making a browser check (especially as there was no way to dismiss it and it was on every step of the form).

![A screenshot of the add browser check form. The section title has a Private preview badge to the right of it.](https://github.com/user-attachments/assets/89d26f67-71f4-47a6-93c3-a814ff38492b)

Reply:
Love the improvement!